lib/streamlit/elements/deck_gl_json_chart.py

```diff
@@ -166,6 +166,8 @@ def pydeck_chart(
         pydeck_obj: Deck | None = None,
         *,
         use_container_width: bool = False,
+        width: int | None = None,
+        height: int | None = None,
```

Comment:
is there a way to abstract out these common types between the various overloads so that we can prevent missing adding new types to these in the future? i won't be shocked if the answer is no given my experience with mypy :(

Reply:
That's a great question and at this moment in time, I'm not sure! I am also not a MyPy expert and not sure of best practices here yet. For now, I would like to keep this change as small as possible, but this is something I'd be happy to look at generally across our types in the future!

Reply:
sounds good to me